src/components/Pagination/Pagination.tsx

```diff
@@ -1 +1,85 @@
-export const Pagination = () => {};
+import classNames from 'classnames';
+
+type Props = {
+  total: number;
+  perPage: number;
+  currentPage: number;
+  setCurrentPage: (value: number) => void;
+};
+
+export const Pagination: React.FC<Props> = ({
+  total,
+  perPage,
+  currentPage,
+  setCurrentPage,
+}) => {
+  const countPagintion: number = Math.ceil(total / perPage);
+
+  const handlePageClick = (page: number) => {
+    setCurrentPage(page);
+  };
+
+  const plassPageClick = () => {
+    if (currentPage < countPagintion) {
+      setCurrentPage(currentPage + 1);
+    }
+  };
+
+  const minusPageClick = () => {
+    if (currentPage > 1) {
+      setCurrentPage(currentPage - 1);
+    }
+  };
+
+  return (
+    <ul className="pagination">
+      <li
+        className={classNames('page-item', {
+          disabled: currentPage === 1,
+        })}
+      >
+        <a
+          data-cy="prevLink"
+          className="page-link"
+          href="#next"
+          aria-disabled={currentPage === 1}
```

Comment:
Create variables for condition that you check twice or more times such as `currentPage === 1`